packages/castmill/config/runtime.exs

```diff
@@ -20,7 +22,11 @@ if System.get_env("PHX_SERVER") do
   config :castmill, CastmillWeb.Endpoint, server: true
 end
 
+IO.puts("Loading runtime configuration for #{config_env()} environment")
+
 if config_env() == :prod do
+  IO.puts("Loading production configuration...")
```

Comment:
Do we want to keep this or was it just for debugging?

Reply:
I think it is good to have actually. We cannot use Logger in runtime.exs so thats why I had to use IO.puts